tests/functional/generate_pages.py

```diff
@@ -69,6 +69,13 @@
 
 """
 
+RELATIONSHIP_RADIO_CLICKER = r"""  click{optionName}() {
+    browser.element('[id="{optionId}"]').click().pause(300)
```

Comment:
Your right, it's the risk. I think it's low risk enough and the chances it needs to change are low enough to hold fire as it will require some effort from @hamishtaplin